src/resources/postcss/base/forms/_selects.pcss

```diff
@@ -0,0 +1,70 @@
+.tribe-events {
+	/* -----------------------------------------------------------------------------
+	 *
+	 * Form Control: Selects
+	 *
+	 * Example:
+	 * <div class="tribe-common-form-control-select">
+	 *   <button aria-haspopup="listbox" aria-labelledby="selectButton" id="selectButton">Option One</button>
+	 *   <ul tabindex="-1" role="listbox">
+	 *     <li id="optionOne" role="option">Option One</li>
+	 *     <li id="optionTwo" role="option">Option Two</li>
+	 *   </ul>
+	 * </div>
+	 *
+	 * ----------------------------------------------------------------------------- */
+
+	 /**
+	  * @todo: implement listbox JS
+		* https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-collapsible.html
```

Comment:
Are you planning to use this lib and the corresponding markup above to power the filter bar view selector, @paulmskim?

Reply:
Yes, I'm thinking a trimmed-down version. The version on w3 has a bunch of customizations (multi-select capability, etc) and I don't think we need that extra JS.